R/user_info_response.R

```diff
@@ -22,14 +22,18 @@ https://data.world"
 #' @seealso \code{\link{get_dataset}}
 user_info_response <- function(structure) {
   required_elements <- get_required_elements()
-  optional_elements <- character()
+  optional_elements <- get_optional_elements()
   me <- c(structure[required_elements], structure[optional_elements])
   class(me) <- "user_info_response"
   return(check_user_info_response(me))
 }
 
 get_required_elements <- function() {
-  c("avatarUrl", "displayName", "id", "created", "updated")
+  c("id", "created", "updated")
+}
+
+get_optional_elements <- function() {
```

Comment:
OK.  This would mean changes to [dataset_, project_, file_, and user_info]_summary_response objects, some of which were present in prior versions of the API wrappers.  Before I remove all those, just want to confirm that's what you want.

Reply:
Yes, let's remove any and all validations on `_response` objects.

Reply:
Done